src/core.js

```diff
@@ -624,8 +624,16 @@ $.extend( $.validator, {
 					return i;
 				} ).length,
 				dependencyMismatch = false,
-				val = this.elementValue( element ),
-				result, method, rule;
+				val, result, method, rule;
+
+			// If a normalizer is defined for this element, then
+			// call it to retreive the changed value instead
+			// of using the real one.
+			val = typeof rules.normalizer === "function" ? rules.normalizer.call( this, element ) : this.elementValue( element );
```

Comment:
> Not using a ternary could ease reading

Should I use an `if` statement instead?

> Do we need to make sure that the normalizer returns a string?

The value can be of any type, I guess.